### PR TITLE
Hex metadata

### LINKED ISSUE #8: Hex package?
I started using Stillir recently and I like it a lot. I'm running inside of Docker containers and passing run-time configuration as environment variables is very handy there. Fetching environment variables is usually clunky, but I've found that a `stillir:set_config/2` block is quite pretty.

I didn't see it up on Hex.pm. How about adding it?

### PR DESCRIPTION
**Pull request by waisbrot**
Also a little rebar3 side-stuff. 

I pushed the package up to Hex, which closes #8 and I added @ferd as an owner.

### CODE REVIEW

File: src/stillir.app.src [hunk 1]
```diff
@@ -2,8 +2,11 @@
 {application, stillir,
  [
   {description, "Read Unix env vars into application config"},
-  {vsn, "1.0.0"},
+  {vsn, git},
   {registered, []},
   {applications, [kernel,
-                  stdlib]}
+                  stdlib]},
+  {maintainers, ["Fred Hebert"]},
```
Comment (ypaq) [inline src/stillir.app.src]: Fred Hebert isn't with Heroku any more. Can you change this to include Jamu Kakar (hex user: jkakar) and Tilman Holschuh (hex user: ypaq).

File: src/stillir.app.src [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
+  {licenses, ["BSD"]},
```
Comment (ypaq) [inline src/stillir.app.src]: I believe hex.pm will automatically extract this information from the `LICENSE` file.

Comment (waisbrot) [reply src/stillir.app.src]: Hex documentation says that the `licenses` field is required: https://hex.pm/docs/rebar3_publish

**Comment by waisbrot** [comment]
@ypaq I changed the maintainer names and added both you and Jamu Kakar as owners of package on Hex

**Comment by ypaq** [comment]
@waisbrot thank you!

**Pull request merged**